eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -17,7 +17,13 @@ class PackageProps
     )
     {
         $this.pkgName = $pkgName
-        $this.pkgVersion = [AzureEngSemanticVersion]::ParseVersionString($pkgVersion)
+
+        $versionObj = [AzureEngSemanticVersion]::ParseVersionString($pkgVersion)
+        $this.pkgVersion = $versionObj.RawVersion
+
+        Write-Host $versionObj.RawVersion
+        Write-Host $this.pkgVersion
```

Comment:
This is because our implementation of ToString is incorrectly putting it into this format https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/SemVer.ps1#L79. We probably should return RawVersion from our ToString implementation.

Reply:
Actually it might not be as simple as that because things like increment will not get us the correct value. We will have to do a little more thinking to figure out the correct way to handle this.